packages/react/src/primitives/types/theme.ts

```diff
@@ -113,20 +113,20 @@ type OverlayKeys =
   | 'overlay.90';
 
 type BrandColorKeys =
```

Comment:
Thinking we should rename this to `PrimaryColorKey` since brand no longer has relevancy

Reply:
+1

Reply:
Should we have Primary and SecondaryColorKey though?

Reply:
@reesscot @calebpollman Just split that into PrimaryColorKeys and SecondaryColorKeys and updated changeset.